testflo/main.py

```diff
@@ -144,6 +144,9 @@ def dir_exclude(d):
                 return True
         return False
 
+    # set this so code will know when it's running under testflo
```

Comment:
Should you restore the env var after testflo finishes? (Or is this being set on the spawned processes?)

Reply:
The environment variable appears not to persist even if not explicitly removed.  For example, for this python code

```
import os
os.environ['FOOBAZ'] = '1'
print("FOOBAZ =", os.environ.get('FOOBAZ'))
```

After running this code and checking the current environment, FOOBAZ isn't found.